brig/cmd/brig/commands/run.go

```diff
@@ -120,20 +120,18 @@ func (a *scriptRunner) send(projectName string, data []byte) error {
 		ProjectID: brigade.ProjectID(projectName),
 		Type:      a.event,
 		Provider:  "brigade-cli",
-		Commit:    a.commit,
-		Payload:   a.payload,
-		Script:    data,
+		Revision: &brigade.Revision{
+			Ref: a.commit,
+		},
+		Payload: a.payload,
+		Script:  data,
 	}
 
 	if err := a.store.CreateBuild(b); err != nil {
 		return err
 	}
 
-	shortRef := b.Commit
-	if len(shortRef) > 8 {
-		shortRef = shortRef[0:8]
-	}
-	podName := fmt.Sprintf("brigade-worker-%s-%s", b.ID, shortRef)
+	podName := fmt.Sprintf("brigade-worker-%s", b.ID)
```

Comment:
Sounds OK becuase I have never relied on the trailing shortRef in the podName.

Maybe It wasn't necessary at all?

Reply:
The shortRef was causing problems in the cases where it was not a valid Kubernetes DNS string. And since a ULID is sortable by time, this made the whole thing more compact. So I like this change because it fixes a bug while also making our use of ULID a little more uniform.